docs/designers-developers/developers/tutorials/javascript/extending-the-block-editor.md

```diff
@@ -59,7 +61,7 @@ function myguten_stylesheet() {
 add_action( 'enqueue_block_assets', 'myguten_stylesheet' );
 ```
 
-Now when you view in the editor and published, you will see your Fancy Quote style, a delicious tomato color text:
+Now when you view in the editor and publish, you will see your Fancy Quote style = a delicious tomato color text:
```

Comment:
Hi @yodiyo, I think using equal in the middle of the sentence may be unexpected in this case I think the comma that we had may make sense. Regarding the change from published to publish I agree with it 👍